### PR TITLE
Preparing release v1.2.0

### PR DESCRIPTION
**Pull request by everett980**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,41 @@
 
 ## Next (unreleased)
 
+## v1.2.0 (May 14, 2019)
+
+### Enhancements
+
+* **Trace detail:** Limit the thickness of spans in the minimap ([@rubenvp8510](https://github.com/rubenvp8510) in [#372](https://github.com/jaegertracing/jaeger-ui/pull/372))
+
+* **Trace detail:** Improve display of long operation names in Operations list ([@kinghuang](https://github.com/kinghuang) in [#351](https://github.com/jaegertracing/jaeger-ui/pull/351))
```
Comment (tiffon) [inline CHANGELOG.md]: I think this is on the search page.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,9 +2,41 @@
...
+
+* **UI Find:** Scroll to first match on load or on press of new locate icon ([@everett980](https://github.com/everett980) in [#367](https://github.com/jaegertracing/jaeger-ui/pull/367))
```
Comment (tiffon) [inline CHANGELOG.md]: Nit: Please use lowercase for follow-on words: `UI find` akin to `Trace detail`

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,9 +2,41 @@
 
 ## Next (unreleased)
```
Comment (tiffon) [inline CHANGELOG.md]: I don't think you need this `Next (unreleased)` line?

File: CHANGELOG.md (hunk 2 above)
Comment (everett980) [reply CHANGELOG.md]: SGTM

File: CHANGELOG.md (hunk 3 above)
Comment (everett980) [reply CHANGELOG.md]: :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (everett980) [reply CHANGELOG.md]: whoops yeah, updated

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,10 +1,40 @@
 # Releases
 
-## Next (unreleased)
+## v1.2.0 (May 14, 2019)
+
+### Enhancements
+
+* **Trace detail:** Limit the thickness of spans in the minimap ([@rubenvp8510](https://github.com/rubenvp8510) in [#372](https://github.com/jaegertracing/jaeger-ui/pull/372))
+
+* **UI find:** Scroll to first match on load or on press of new locate icon ([@everett980](https://github.com/everett980) in [#367](https://github.com/jaegertracing/jaeger-ui/pull/367))
```
Comment (yurishkuro) [inline CHANGELOG.md]: I suggest renaming "UI find" to "Search in trace"

Comment (everett980) [reply CHANGELOG.md]: @yurishkuro we've been trying to separate the terms "Find" and "Search" as "find in loaded data" vs "request new data"
Also "in trace" isn't always the right label as there are multiple traces for a diff, and while DDG data is generated from traces, it isn't trace data.

**Pull request merged**